filters/src/voxel_grid_label.cpp

```diff
@@ -37,8 +37,19 @@
  */
 
 #include <map>
+#include <pcl/common/common.h> // for getMinMax3D
 #include <pcl/filters/voxel_grid_label.h>
-#include <pcl/filters/impl/voxel_grid.hpp>
+#include <boost/mpl/size.hpp> // for boost::mpl::size
+
+struct point_index_idx
+{
+  unsigned int idx;
```

Comment:
Minor thing but the guide says the fields should be suffixed with underscore:
https://pcl.readthedocs.io/projects/advanced/en/latest/pcl_style_guide.html#member-variables
Instead of the input paramters in the constructor 😄 ?

Reply:
Could we give idx a better name:
idx = centroid leaf index according to: https://github.com/PointCloudLibrary/pcl/blob/b0b25194f214112e3eb3db876f33960fd85f5794/filters/include/pcl/filters/impl/voxel_grid.hpp#L275
and
https://github.com/PointCloudLibrary/pcl/blob/b0b25194f214112e3eb3db876f33960fd85f5794/filters/include/pcl/filters/impl/voxel_grid.hpp#L323

Should we move it to some common header in pcl/filters/xx.h - so it can be used in both voxel_grid and voxel_grid_label?

Reply:
I guess :smile: - lets just move it then to voxel_grid.h in a namespace.
I don't mind, but we can also leave it as is.

Reply:
I moved it to voxel_grid.h. I would prefer to not change any field/parameter names in this PR.